cmake/ElektraUninstall.cmake

```diff
@@ -11,7 +11,12 @@ endif (NOT EXISTS "${MANIFEST}")
 
 #message (MANIFEST IS ${MANIFEST})
 
+# =========
+# = Files =
+# =========
+
 file (READ "${MANIFEST}" files)
+string (APPEND files "\n/usr/local/lib/python2.7/site-packages/elektra_gen-${KDB_VERSION}-py2.7.egg-info")
```

Comment:
Seems to ignore CMAKE_INSTALL_PREFIX? And also python version? (Please look into the swig binding how the path is concatinated)

Reply:
> And also python version? (Please look into the swig binding how the path is concatenated)

The problem should be fixed in my latest update. Thank you for the helpful input.